### PR TITLE
Add CMake support

### PR DESCRIPTION
**Pull request by matyas-streamhpc**

### CODE REVIEW

File: docs/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# MIT License
+#
+# Copyright (c) 2023 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (Rmalavally) [inline docs/CMakeLists.txt:3]: Please update the copyright to 2023-24.

File: docs/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+# Set minimum required version of cmake
+cmake_minimum_required(VERSION 3.10)
+
+project(Docs)
+
+find_program(SPHINX_EXECUTABLE NAMES sphinx-build)
+
+if (NOT SPHINX_EXECUTABLE)
+    message(FATAL_ERROR "The sphinx-build not found.")
```
Comment (Rmalavally) [inline docs/CMakeLists.txt:31]: Change error to -

"Sphinx build not found."

File: docs/CMakeLists.txt (hunk 1 above)
Comment (saadrahim) [reply docs/CMakeLists.txt:3]: It isn't 2024 yet.

File: cmake/Modules/utils.cmake
Comment (saadrahim) [inline cmake/Modules/utils.cmake:1]: LIkely not needed in this project

File: cmake/Modules/env.cmake
Comment (saadrahim) [inline cmake/Modules/env.cmake:1]: Likely not needed in this project.

File: CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+cmake_minimum_required(VERSION 3.18.0)
+
+project(ROCm VERSION 5.7.1 LANGUAGES NONE)
```
Comment (saadrahim) [inline CMakeLists.txt:25]: Let's make one source of truth for the version number

**Review by saadrahim — APPROVED** [review]

File: cmake/Modules/env.cmake
Comment (matyas-streamhpc) [reply cmake/Modules/env.cmake:1]: It is not needed. Removed.

File: cmake/Modules/utils.cmake
Comment (matyas-streamhpc) [reply cmake/Modules/utils.cmake:1]: It is not needed. Removed.

File: docs/CMakeLists.txt (hunk 2 above)
Comment (matyas-streamhpc) [reply docs/CMakeLists.txt]: The CMakeLists.txt was updated based on the ROCprofiler. The mentioned section is removed.

**Comment by saadrahim** [comment]
Please update the changelog and README.md to reflect these changes.

File: tools/autotag/templates/rocm_changes/6.0.0.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<!-- markdownlint-disable first-line-h1 -->
+<!-- markdownlint-disable no-duplicate-header -->
+
+### What's new in this release
+
+#### Documentation
+
+CMake support is added for the documentation.
```
Comment (saadrahim) [inline tools/autotag/templates/rocm_changes/6.0.0.md:8]:
```suggestion
CMake support is added for the documentation in the [ROCm repository](https://github.com/RadeonOpenCompute/ROCm).
```

**Review by saadrahim — APPROVED** [review]

**Pull request merged**